ansible/roles/minidlna/defaults/main.yml

```diff
@@ -0,0 +1,147 @@
+minidlna__base_packages: [ 'minidlna' ]
+
+                                                                   # ]]]
+# .. envvar:: minidlna__packages [[[
+#
+# List of additional APT packages to install during :command:`minidlna`
+# configuration.
+minidlna__packages: []
+
+                                                                   # ]]]
+                                                                   # ]]]
+# MiniDLNA configuration [[[
+# ----------------------
+
+# .. envvar:: minidlna__http_port [[[
+#
+# Port for HTTP traffic. Defaults to 8200.
+minidlna__http_port: 8200
+
+                                                                   # ]]]
+# .. envvar:: minidlna__media_dirs [[[
+#
+# List of media directories to scan and present to end user.
+#
+# If you want to restrict a media_dir to a specific content type, you can
+# prepend the directory name with a letter representing the type (A, P or V),
+# followed by a comma, as so:
+#
+# * "A" for audio    (eg. media_dir=A,/var/lib/minidlna/music)
+# * "P" for pictures (eg. media_dir=P,/var/lib/minidlna/pictures)
+# * "V" for video    (eg. media_dir=V,/var/lib/minidlna/videos)
+# * "PV" for pictures and video (eg. media_dir=PV,/var/lib/minidlna/digital_camera)
+#
+minidlna__media_dirs: []
+
+                                                                   # ]]]
+# .. envvar:: minidlna__default_configuration [[[
+#
+# YAML dictionary of {name, value} for configuration file.
+minidlna__default_configuration:
+  - name: "inotify"
+    value: "yes"
+                                                                   # ]]]
+# .. envvar:: minidlna__configuration [[[
+#
+# YAML dictionary of {name, value} for configuration file.
+minidlna__configuration: []
+
+                                                                   # ]]]
+# .. envvar:: minidlna__default_album_art_names [[[
+#
+# List of file names to look for when searching for album art.
+# Names should be delimited with a forward slash ("/").
+minidlna__default_album_art_names:
+  - "Cover.jpg/cover.jpg/AlbumArtSmall.jpg/albumartsmall.jpg"
+  - "AlbumArt.jpg/albumart.jpg/Album.jpg/album.jpg"
+  - "Folder.jpg/folder.jpg/Thumb.jpg/thumb.jpg"
+                                                                   # ]]]
+# .. envvar:: minidlna__album_art_names [[[
+#
+# List of file names to look for when searching for album art.
+# Names should be delimited with a forward slash ("/").
+minidlna__album_art_names: []
+
+                                                                   # ]]]
+# .. envvar:: minidlna__allow [[[
+#
+# List of IP addresses or CIDR subnets which are allowed to connect to the
+# MiniDLNA instances over the network, on all hosts in the Ansible
+# inventory. This variable configures the firewall for all instances at the
+# same time, for individual instance configuration you should modify the
+# :envvar:`minidlna__ferm__dependent_rules` variable directly.
+minidlna__allow: {}
```

Comment:
This should be a list (`[]`) and not a YAML dictionary (`[]`), otherwise the `ferm` role will not parse this correctly.